src/test/java/org/simulator/sedml/SEDMLExecutorTest.java

```diff
@@ -181,14 +181,12 @@ public final void testRepeatedScanOscli() throws XMLException, OWLOntologyCreati
     }
 
     @Test
-    @Ignore
     public final void testRepeatedSteadyScanOscli() throws XMLException, OWLOntologyCreationException, IOException {
         String resource = "/sedml/L1V2/repeated-steady-scan-oscli/repeated-steady-scan-oscli.xml";
         testSpecificationExample(resource);
     }
 
     @Test
-    @Ignore
     public final void testRepeatedStochasticRuns() throws XMLException, OWLOntologyCreationException, IOException {
         String resource = "/sedml/L1V2/repeated-stochastic-runs/repeated-stochastic-runs.xml";
```

Comment:
Please extract this to a constant.

Reply:
Here, the resources are provided for all the tests separately (in their corresponding test methods) so, I don't this to make this as a constant.